pages/index.html

```diff
@@ -4,9 +4,31 @@
   <meta charset="UTF-8">
```

Comment:
Generally, the index.html page would go at the top level of the site (not within a subdirectory). When we visit a website (e.g., www.github.com) when no page is part of the URL, the web server will look for one of a few default file names (index.html is usually among them) and serve the first one it finds. While a web server can be configured to serve a default file from elsewhere (such as under the pages folder) it's often easier to have the index.html live in the site root, and then potentially have the other pages stored in other locations. This can affect the paths required for locating style files, or other pages in anchor tags.